bazel/get_workspace_status

```diff
@@ -40,3 +40,6 @@ git diff-index --quiet HEAD -- || {
 }
 echo "BUILD_SCM_STATUS ${tree_status}"
 echo "STABLE_BUILD_SCM_STATUS ${tree_status}"
+
+git_version=$(git describe --tags --dirty --always --match 'v[0-9]*' | cut -c 2-)
```

Comment:
Note: matches the proposed format [here](https://github.com/10gen/envoy-serverless/blob/develop/evergreen/archive-evergreen-rhel7-amd64.sh#L12)

Reply:
Could you please add a comment referring to the file you are linking here?